tensorflow_datasets/scripts/download_and_prepare.py

```diff
@@ -42,13 +42,13 @@
 import os
 import pdb
 import time
+import termcolor
```

Comment:
Thanks for fixing this, merging it now interally so should be merged on GitHub soon.
However not sure why you changed the import order as termcolor isn't part of the standard libraries.